### PR TITLE
update

### PR DESCRIPTION
**Pull request by fomod-in**
new texts following UI updates

### CODE REVIEW

**Comment by jacogr** [comment]
Looks good, some conflicts did sneak in, so a merge is just required.

**Comment by fomod-in** [comment]
you can merge it

Sent from [ProtonMail](https://protonmail.com)

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
Il giovedì 8 ottobre 2020 12:42, Jaco Greeff <notifications@github.com> ha scritto:

File: packages/apps/public/locales/it/translation.json [hunk 1]
```diff
@@ -890,7 +899,7 @@
...
   "You don't have any accounts. Some features are currently hidden and will only become available once you have accounts.": "Non hai nessun account. Alcune funzioni sono attualmente nascoste e diventeranno disponibili solo una volta che si dispone di un account.",
-  "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app on the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.": "Hai {{claimCount}}  accounts che necessitano di verifica. Usa la app di Riscatto Tokens sul menu di navigazione in alto per completare il processo. Fin quando non lo completerai, i saldi di questi account non verranno visualizzati.",
+  "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app on the left navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.": "Hai {{claimCount}}  accounts che necessitano di verifica. Usa la app di Riscatto Tokens sul menu di navigazione in alto per completare il processo. Fin quando non lo completerai, i saldi di questi account non verranno visualizzati.",
```
Comment (jacogr) [inline packages/apps/public/locales/it/translation.json:902]:
```suggestion
  "You have {{claimCount}} accounts that need attestations. Use the Claim Tokens app on the navigation bar to complete the process. Until you do, your balances for those accounts will not be reflected.": "Hai {{claimCount}}  accounts che necessitano di verifica. Usa la app di Riscatto Tokens sul menu di navigazione in alto per completare il processo. Fin quando non lo completerai, i saldi di questi account non verranno visualizzati.",
```

File: packages/apps/public/locales/it/translation.json [hunk 2]
```diff
@@ -591,7 +600,7 @@
   "The account you will send funds from.": "L'account mittente dei fondi",
   "The account's password specified at the creation of this account.": "La password dell'account specificata al momento della creazione",
   "The actual JSONRPC module and function to make a call to.": "Il modulo JSONRPC attuale e la funzione da richiamare",
-  "The actual account you wish to submit the bid with": "L' account dal quale desideri inoltrare l'offerta",
+  "The actual account you wish to sub,it the bid with": "L' account dal quale desideri inoltrare l'offerta",
```
Comment (jacogr) [inline packages/apps/public/locales/it/translation.json:603]:
```suggestion
  "The actual account you wish to submit the bid with": "L' account dal quale desideri inoltrare l'offerta",
```

**Comment by jacogr** [comment]
Thank you!

**Pull request merged**